Add permissions required to README.md (#64)
yuzutech/annotations-action: README.md

]
```

## Permissions

It is possible you have a `GITHUB_TOKEN` with restricted access by default, in which
case you might get the following error:

```
Error: GitHubApiUnauthorizedError: Unable to create a check, please make sure that
the provided 'repo-token' has write permissions to 'your/repo' - cause:
HttpError: Resource not accessible by integration
```

You need to provide the `checks` write permission:
```yaml
permissions:
  checks: write
```

> Note that setting a `permissions` field will disable all omitted permissions, like

Reviewer: ```suggestion
You can use `permissions` either as a top-level key, to apply to all jobs in the workflow, or within specific jobs. When you add the permissions key within a specific job, all actions and run commands within that job that use the `GITHUB_TOKEN` gain the access rights you specify.

**IMPORTANT**: Setting a `permissions` field will disable all omitted permissions, like
```

> the `contents: read`, which is needed to clone and read your repository.

Reviewer: ```suggestion
the `contents: read`, which is needed to clone and read your repository.

You can learn more about `GITHUB_TOKEN` permissions at: https://docs.github.com/en/actions/security-guides/automatic-token-authentication#permissions-for-the-github_token
```

]
```

## Permissions

It is possible you have a `GITHUB_TOKEN` with restricted access by default, in which case you might get the following error:

```
Error: GitHubApiUnauthorizedError: Unable to create a check, please make sure that the provided 'repo-token' has write permissions to 'your/repo' - cause: HttpError: Resource not accessible by integration
```

> You need to provide the `checks` write permission:
> ```yaml

Reviewer: ```suggestion
You need to provide the `checks` write permission:

```yaml
```

]
```

## Permissions

> It is possible you have a `GITHUB_TOKEN` with restricted access by default, in which
> case you might get the following error:

Reviewer: ```suggestion
It is possible you have a `GITHUB_TOKEN` with restricted access by default, in which case you might get the following error:
```

```

> Error: GitHubApiUnauthorizedError: Unable to create a check, please make sure that
> the provided 'repo-token' has write permissions to 'your/repo' - cause:
> HttpError: Resource not accessible by integration

Reviewer: ```suggestion
Error: GitHubApiUnauthorizedError: Unable to create a check, please make sure that the provided 'repo-token' has write permissions to 'your/repo' - cause: HttpError: Resource not accessible by integration
```

```

You need to provide the `checks` write permission:
```yaml
permissions:
  checks: write
```

> You can use `permissions` either as a top-level key, to apply to all jobs in the workflow, or within specific jobs. When you add the permissions key within a specific job, all actions and run commands within that job that use the `GITHUB_TOKEN` gain the access rights you specify.

Reviewer: ```suggestion
You can use `permissions` either as a top-level key, to apply to all jobs in the workflow, or within specific jobs.
When you add the permissions key within a specific job, all actions and run commands within that job that use the `GITHUB_TOKEN` gain the access rights you specify.
```
